.gitignore

```diff
@@ -10,4 +10,6 @@
 # do ignore web-vitals/dist contents
 **/node_modules/web-vitals/dist/*
 # don't ignore web-vitals.min.js
-!**/node_modules/web-vitals/dist/web-vitals.es5.min.js
\ No newline at end of file
+!**/node_modules/web-vitals/dist/web-vitals.es5.min.js
+
+src/browser_action/secrets.js
```

Comment:
To avoid leaking the CrUX API key in GitHub, I stored it in this `secrets.js` file, which is ignored by git.

The contents of the file are simply:

```js
export const CRUX_API_KEY = 'API_KEY';
```

Reply:
I think this approach is a relatively lightweight way to avoid committing `API_KEY` to this repo while still allowing releases to be published to CWV. Based on our last discussion, we may proceed with publishing the key within the distributed package but I'm anticipating the risk of this to be low. That said, will check with the Chrome extensions team in case they have thoughts :)